File: 1.FDP/2° Competencia/C++/Repeticion/Ejercicio5.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(){
	cout << "Convertir pesos a dolares.\n";
	double Pesos, ValorDolar, Dolares;
	cout << "Se repetira 3 veces.\n";
	cout << "\n";
	for ( int i=0; i<3; i++) {
		cout << "Ingresar la cantidad de pesos:";
		cin >> Pesos;
		if (Pesos > 0) {
			cout << "Ingresar el valor del dolar:";
			cin >> ValorDolar;
			if (ValorDolar > 0) {
				Dolares= Pesos/ValorDolar;
				cout << "Cantidad de dolares = " << Dolares << "\n";
				cout << "\n";
			} else {
				cout << "Error del valor del dolar.\n";
				cout << "///////////////////////.\n";
			}
		} else {
			cout << "Error de los pesos.\n";
			cout << "///////////////////////.\n";
			
		}
	}		
	
	return 0;
}

File: 1.FDP/2° Competencia/Java/Repeticion/Ejercicio5.java
import java.util.Scanner;

public class Ejercicio5 {
	public static void main (String[] args){
		Scanner scan= new Scanner(System.in);
		System.out.println("Calcular la convercion de pesos a dolares.\n");
		double Pesos, Dolares, Convercion;
		int Respuesta;
		System.out.println("Cuantas veces deceas que se repita:");
		Respuesta= scan.nextInt();
		for (int i=0; i<Respuesta; i++) {
			System.out.println("Ingresar la cantidad de pesos:");
			Pesos= scan.nextDouble();
			if (Pesos > 0) {
				System.out.println("Ingresa el valor del dolar:");
				Dolares= scan.nextDouble();
				if (Dolares > 0) {
					Convercion= Pesos/Dolares;
					System.out.println("Cantidad de dolares= " + Convercion);
					System.out.println("//////////////////////////////////////.");
				} else {
					System.out.println("Error en el valor del dolar");
					System.out.println("////////////////////////////////");
				}	
			} else {
				System.out.println("Error de pesos");
				System.out.println("//////////////////////////");
			}	
		}
	}
}